packages/app/src/pages/StakeDashboard.tsx

```diff
@@ -174,21 +176,53 @@ export const StakeDashboard: FC = () => {
       </Panel>
       <div className="grid gap-8 grid-cols-3 grid-rows-1 my-10 text-base">
         <InfoBox className="p-2 rounded text-center">
-          <span className="font-bold text-xl">0.00</span>
+          <span className="font-bold text-xl">
+            {stakeBalance &&
+              toFixedWithPrecision(toSol(stakeBalance.extractableYield))}
+          </span>
           <br />
-          Earned
+          gSOL
         </InfoBox>
         <InfoBox className="p-2 rounded text-center">
-          <span className="font-bold text-xl">0.00</span>
+          <span className="font-bold text-xl">
+            {stakeBalance &&
+              toFixedWithPrecision(
+                solToCarbon(toSol(stakeBalance.extractableYield))
+              )}
+          </span>
           <br />
           tCO₂E
         </InfoBox>
         <InfoBox className="p-2 rounded text-center">
-          <span className="font-bold text-xl">0.00</span>
+          <span className="font-bold text-xl">
+            {treasuryBalanceLamports &&
+              stakeBalance &&
+              toFixedWithPrecision(
+                solToCarbon(
+                  toSol(
+                    treasuryBalanceLamports.add(stakeBalance.extractableYield)
+                  )
+                )
+              )}
+          </span>
           <br />
           Total tCO₂E
         </InfoBox>
       </div>
+
+      <div className="flex flex-col items-center">
+        {delayedUnstakeTickets.map((ticket) => {
+          return (
+            <WithdrawTicket
+              key={ticket.address.toBase58()}
+              ticket={ticket}
+              redeem={redeem}
+            />
+          );
+        })}
+        {/* <WithdrawTicket />
```

Comment:
Should this be removed?

Reply:
Yes, I will do it. It was just for testing.